FreshAir/UI/RZFReleaseNoteCollectionViewCell.m

```diff
@@ -90,9 +133,18 @@ - (void)setupInfoContainerView
                                                      attribute:NSLayoutAttributeBottom
                                                     multiplier:1.0f
                                                       constant:kRZFInfoContainerViewTopPadding]];
-    
-    self.infoContainerView.backgroundColor = [UIColor rzf_defaultReleaseNoteTintColor];
-    
+
+    if ( self.releaseNotes.accentColor ) {
+        RZFAlphaColor *color = self.releaseNotes.accentColor;
+        self.infoContainerView.backgroundColor = [UIColor colorWithRed:color.red.floatValue/255.0f
```

Comment:
I'd move the UIColor creation into the RZFAlphaColor, this  method is duplicated in a few places. Also you may be able to just use UIColor and minimize the exposure of RZFAlphaColor.